eng/testing/RunnerTemplate.sh

```diff
@@ -230,40 +169,30 @@ if [[ "$(uname -s)" == "Linux" && $test_exitcode -ne 0 ]]; then
 
     have_sleep=$(which sleep)
     if [ -x "$have_sleep" ]; then
-      echo Waiting a few seconds for any dump to be written..
+      echo Waiting a few seconds for any dump to be written...
       sleep 10s
     fi
   fi
 
-  echo cat /proc/sys/kernel/core_pattern: $(cat /proc/sys/kernel/core_pattern)
```

Comment:
I'm going to have to bring this back. The names of the dump files vary too much and some don't even end in *.dmp. The XUnitLogChecker tool is limited to a very narrow wildcard, which I expanded in this PR, but it's not going to be enough. I'll reuse this code I removed but will adapt it so that the found coredumps are renamed to include the extension.

Reply:
Can't you name the dumps when you set the variables that turn them on?

Reply:
You mean `DOTNET_DbgMiniDumpName=$EXECUTION_DIR/coredump.%d.dmp`? I already have that up there but does not seem to work for all cases.